src/main/java/com/github/manolo8/darkbot/utils/ApiErrors.java

```diff
@@ -26,7 +26,7 @@ private static String getMessage(BrowserApi api) {
             return "You're using a <strong>removed API</strong>, use the recommended API!";
         if (api == BrowserApi.NO_OP_API)
             return "Error loading the no-op api, how did this happen?";
-        if (api != BrowserApi.KEKKA_PLAYER)
+        if (api != BrowserApi.TANOS_API)
```

Comment:
```suggestion
        if (api != (OSUtil.isLinux() ? BrowserApi.TANOS_API : BrowserApi.KEKKA_PLAYER))
```

maybe `OSUtil.getDefaultAPI` that just does this logic and re-use it in the 3 places it's used at?